### PR TITLE
Adding the code review by waa94

### PR DESCRIPTION
**Pull request by Rymouxi**

### CODE REVIEW

File: thorgame/ro315/Code review by waa94.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# ### Power of Thor - Episode 1
```
Comment (bjones1) [inline thorgame/ro315/Code review by waa94.py:1]: Delete this file from your PR.

File: thorgame/ro315/Modified code after review.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# ## Power of Thor - Episode 1
+#
+# **Link to the problem :
+# [CodinGame Power of Thor](https://www.codingame.com/training/easy/power-of-thor-episode-1)**
+#
+# ### **Problem Overview:**
+#
+# <span class="hljs-comment">In this puzzle, Thor needs to move toward the
```
Comment (bjones1) [inline thorgame/ro315/Modified code after review.py:8]: Remove hilights from the reviewer.

File: thorgame/ro315/Modified code after review.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    else:
+     # No movement in the X direction if aligned.
+        direction_x = "" 
+
+    # ### Condition on Y:
+    #
+    # Thor moves North or South depending on the position of `light_y` relative
+    # to `initial_ty`.
+    if light_y > initial_ty:
+    # Move South if the light is below Thor.
```
Comment (bjones1) [inline thorgame/ro315/Modified code after review.py:49]: Align comments with the following line of source code.

**Review by bjones1 — CHANGES_REQUESTED** [review]

File: thorgame/ro315/Code review by waa94.py (hunk 1 above)
Comment (Rymouxi) [reply thorgame/ro315/Code review by waa94.py]: File deleted and change commited to the ro315-1 branch

File: thorgame/ro315/Modified code after review.py (hunk 2 above)
Comment (Rymouxi) [reply thorgame/ro315/Modified code after review.py:8]: The highlights from the reviewer weren't in the revised code.

**Comment by Rymouxi** [comment]
All the required changes were applied.

**Pull request merged**